.github/workflows/zephyr.yml

```diff
@@ -254,3 +259,43 @@ jobs:
           path: |
             ${{ github.workspace }}/workspace/build-sof-staging
             ${{ github.workspace }}/workspace/**/compile_commands.json
+
+
+  compare-linux-win:
+
+    runs-on: ubuntu-latest
+
+    # - We don't compare _all_ the builds, and
+    # - even when some of the ones we compare fail, we still want to compare the rest.
+    if: ${{ always() }}
+    needs: [build-linux, build-windows]
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: ./sof
+
+      - name: Download Windows and Linux builds
+        uses: actions/download-artifact@v3
+
+      - name: apt-get dos2unix
+        run: sudo apt-get update; sudo apt-get -y install dos2unix
+
+      - name: Delete and fix expected differences
+        run: |
+          ls -l
+          # run it twice to make sure it's idempotent
```

Comment:
I do mean "idempotent". This makes it easier to test this script locally (testing in Github is excruciatingly slow because you have to wait for all the builds to complete)

Reply:
All right, new English word for me for today! 😄